symbols/arm9.yml

```diff
@@ -9343,6 +9343,71 @@ arm9:
         
         r0: matrix_4x3 pointer
         r1: GXFIFO pointer
+    - name: IncrementThreadCount
+      address:
+        EU: 0x20794E8
+        NA: 0x2079150
+      description: |-
+        Increments thread_info::thread_count by 1 and returns the new value.
+        
+        return: thread_info::thread_count + 1
```

Comment:
This phrasing is ambiguous (does `thread_count` refer to the value before or after the increment?). I think it'd be better to just say something like "return: new thread count".

Reply:
Sounds good